README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### A Little Bit About Me!
+
+My name is Lauren Park. I am a second year at the College, and am a Sociology and French major. I was born and raised in Los Angeles. I am also on the women's varsity tennis team. 
+
+#### These are a few of *my favorite things* to do in Chicago:
+* Eating at new restaurants
+* Running along the lake
+* Exploring new neighborhoods
+* Sitting on the quad when the weather is nice
+
+[Here is the link to one of my favorite food blog guides! Enjoy!:)](https://chicago.eater.com/2018/1/5/16151978/best-food-chicago-restaurants-city-guide)
+
+Last but not least, here is a picture of the river. I think that West Loop is home to some of the **best food** in Chicago downtown! *(Picture is taken from hotspotrentals.com)*
+
+![](https://www.hotspotrentals.com/wp-content/uploads/2018/12/shot-of-West-Loop-1.jpg)
```

Comment:
You could also add a link to the image. 

Code:

[![your image caption](image.png)](www.link_to_image.com)